packages/winston-log-enricher/lib/createFormatter.js

```diff
@@ -11,6 +11,9 @@ module.exports = function createFormatter(newrelic, winston) {
   if (!newrelic.shim) {
     // Continue to log original message with JSON formatter
     return winston.format.json
+  } else if (newrelic.shim.isWrapped(winston.createLogger)) {
+    newrelic.shim.logger.debug('Winston is already instrumented. Skipping enrichment...')
```

Comment:
we may want this to be warn so customers will see it during their normal runs

Reply:
Good idea.